src/pages/Settings/EmailSettings.vue

```diff
@@ -0,0 +1,518 @@
+								Send only legally required emails
+							</option>
+						</kv-dropdown-rounded>
+					</div>
+				</template>
+			</kv-settings-card>
+		</div>
+		<div class="row">
+			<!-- General Settings -->
+			<kv-settings-card
+				class="columns small-12 large-8"
+				title="General Kiva / account updates"
+				:disabled="form.globalUnsubscribed"
+			>
+				<template #content>
+					<form @submit.prevent>
+						<fieldset>
+							<kv-button class="text-link" @click.native="generalToggleAll()">
+								{{ generalAllSelected ? "Deselect" : "Select" }} All
+							</kv-button>
+						</fieldset>
+						<fieldset>
+							<kv-checkbox
+								id="lenderNews"
+								name="lenderNews"
+								v-model="form.lenderNews"
+								class="email-settings__checkbox"
+							>
+								News, inspiration and promotions
+							</kv-checkbox>
+						</fieldset>
+						<fieldset>
+							<kv-checkbox
+								id="accountUpdates"
+								name="accountUpdates"
+								v-model="form.accountUpdates"
+								class="email-settings__checkbox"
+							>
+								Portfolio and balance updates
+							</kv-checkbox>
+						</fieldset>
+						<fieldset>
+							<label for="repayment-updates-input">Repayment notifications</label>
+							<kv-dropdown-rounded
+								id="repayment-updates-input"
+								v-model="form.repaymentUpdates"
+							>
+								<option value="none">
+									Do not send
+								</option>
+								<option value="nightly">
+									Send a nightly digest
+								</option>
+								<option value="monthly">
+									Send a monthly digest
+								</option>
+							</kv-dropdown-rounded>
+						</fieldset>
+						<!-- TODO Implement Service Worker for push notifications -->
+						<!-- When repaymentUpdates == 'none' this should be hidden -->
+						<!-- <div class="input-wrapper nested">
+								<input id="push-repayment-updates-input" type="checkbox">
+								<label for="push-repayment-updates-input">
+									Receive notifications on this device
+								</label>
+							</div> -->
+						<fieldset>
+							<label for="autolend-updates-input">Autolending notifications</label>
+							<kv-dropdown-rounded
+								id="autolend-updates-input"
+								v-model="form.autolendUpdates"
+							>
+								<option value="none">
+									Do not send
+								</option>
+								<option value="nightly">
+									Send a nightly digest
+								</option>
+								<option value="monthly">
+									Send a monthly digest
+								</option>
+							</kv-dropdown-rounded>
+						</fieldset>
+						<fieldset>
+							<a href="/lend/saved-search">Saved searches notifications</a>
+						</fieldset>
+					</form>
+				</template>
+			</kv-settings-card>
+		</div>
+
+		<div class="row">
+			<!-- Loan Settings -->
+			<kv-settings-card
+				class="columns small-12 large-8"
+				title="Lending activity updates"
+				:disabled="form.globalUnsubscribed"
+			>
+				<template #content>
+					<form @submit.prevent>
+						<fieldset>
```

Comment:
There's a lot of fieldsets here. They're usually for grouping a set of related inputs. So for this area, I think you'd have the Select All button and the two checkboxes below in the same fieldset.

Reply:
Or maybe everything in this settings-card is in the same fieldset